addon/controllers/new.js

```diff
@@ -6,7 +6,7 @@ export default Ember.Controller.extend({
     post (title, body) {
       let blog = this.get('blog')
       blog.add(title, body)
-      this.transitionToRoute('/blog')
+      this.transitionToRoute('index')
```

Comment:
I did try that and no there is no route name blog. /blog was going back to the index.